app/controllers/return-logs.controller.js

```diff
@@ -8,9 +8,24 @@
 const Boom = require('@hapi/boom')
 
 const SubmitViewReturnLogService = require('../services/return-logs/submit-view-return-log.service.js')
-
+const DownloadReturnLogService = require('../services/return-logs/download-return-log.service.js')
 const ViewReturnLogService = require('../services/return-logs/view-return-log.service.js')
 
+async function downloadReturnLog(request, h) {
```

Comment:
IMHO we can just called this handler `download`. It's on the return logs controller, so not only can this be assumed, it matches the convention of other handlers (referring to the action and not mentioning the root resource).